test/party-external-rating/get-party-external-ratings.api-test.ts

```diff
@@ -56,6 +56,16 @@ describe('GET /parties/{partyIdentifier}/external-ratings', () => {
     expect(body).toStrictEqual([]);
   });
 
+  it('returns a 401 response when the API Key is missing', async () => {
+    givenAuthenticationWithTheIdpSucceeds();
+    requestToGetExternalRatingsForParty().reply(200, []);
+
+    const { status, body } = await api.getWithoutAuth(getPartyExternalRatingsUrl);
+
+    expect(status).toBe(401);
+    expect(body).toStrictEqual({ message: 'Unauthorized', statusCode: 401 });
+  });
```

Comment:
I think we should also test:
- if the header is correct but the api key is wrong
- if the header is wrong but the api key is correct

We can create a function for injecting these tests in the `test/common-tests` folder and use that function for each endpoint's API tests

Reply:
this is updated
the strategy needs to be present at all times - otherwise supertest won't execute the request